### PR TITLE
Add > SP Cloud > Integrations Doc > Gitpod

### PR DESCRIPTION
**Pull request by RahulSrivastav14**

### CODE REVIEW

File: docs/integrations/gitpod.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+title: Using Steampipe in Gitpod
+sidebar_label: Gitpod
+---
+# Using Steampipe in Gitpod
+
+[Gitpod](https://www.gitpod.io/) is an open source platform provisioning ready-to-code developer environments that integrates with GitHub. Here we integrate a Github project with Gitpod to install Steampipe, then install a plugin and run a query.
+
+## Installing Steampipe in Gitpod
+
+To run scripts, first connect your GitHub repository to Gitpod workspace and create a `.gitpod.yml` file that contains the definitions. Here's an example that installs Steampipe.
```
Comment (judell) [inline docs/integrations/gitpod.md:11]:
```suggestion
To run scripts, first connect your GitHub repository to your Gitpod workspace and create a `.gitpod.yml` file that contains the definitions. Here's an example that installs Steampipe.
```

File: docs/integrations/gitpod.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+      steampipe plugin install steampipe
+      steampipe plugin install rss
+
+ports:
+ # Steampipe/ PostgreSQL
+  - port: 9193
+
+```
+
+<div style={{"marginBottom":"2em","borderWidth":"thin", "borderStyle":"solid", "borderColor":"lightgray", "padding":"20px", "width":"50%"}}>
```
Comment (judell) [inline docs/integrations/gitpod.md:41]: images are missing in the preview here: https://steampipe-m1k1m54w1-turbot.vercel.app/docs/integrations/gitpod

(I found the preview by way of https://cloud.steampipe.io/org/acme/workspace/jon/dashboard/steampipe_stats.dashboard.Vercel)

File: docs/integrations/gitpod.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+<img alt="gitlab-plugin-installed" src="/images/docs/ci-cd-pipelines/gitpod-config-install.png" />
+</div>
+
+Next, we'll update the file with a query to list items from a channel from `rss_item`.
```
Comment (judell) [inline docs/integrations/gitpod.md:45]:
```suggestion
Next, we'll update the file with a query to list items from an RSS feed.
```

File: docs/cloud/integrations/gitpod.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+There are a number of Gitpod extensions that connect to and query databases. For this example we'll use [SQLTools PostgreSQL](https://open-vsx.org/extension/mtxr/sqltools-driver-pg).
+
+To create a connection via the command palette, choose `Add New Connection`, select PostgreSQL and add the connection details.
+
+<div style={{"marginTop":"1em", "marginBottom":"1em", "width":"90%"}}>
+<img src="/images/docs/cloud/gitpod-connect-details.png" />
+</div>
+
+Once you're connected to Steampipe Cloud, the SQLTools Explorer can access the tables available in your workspace. Here we'll use the RSS connection and use this query to list items from a channel. Gitpod previews the data which can be exported into a JSON or CSV format.
```
Comment (judell) [inline docs/cloud/integrations/gitpod.md:46]:
```suggestion
Once you're connected to Steampipe Cloud, the SQLTools Explorer can access the tables available in your workspace. Here we'll use the RSS connection and use this query to list items from a feed. Gitpod previews the data which can be exported into a JSON or CSV format.
```

File: docs/integrations/gitpod.md (hunk 2 above)
Comment (RahulSrivastav14) [reply docs/integrations/gitpod.md:41]: The images were in a different directory. I just changed it. It should show up in the preview now.

**Review by judell — APPROVED** [review]

**Pull request merged**